app/models/user.rb

```diff
@@ -12,6 +16,7 @@ class User < ApplicationRecord
   validates :password, confirmation: true , if: -> { new_record? || changes[:crypted_password] }
 
   before_save :lowercase_email, if: :email =~ /[A-Z]/
+  after_archive :destroy_attenables
```

Comment:
I'm not sure but what I understand is archiving is we can anytime unarchive when needed. user delete(archive) and than when they come back, their account will be unarchived with their data. in this case, we should keep group as well. otherwise, just delete user. if I'm wrong, could you explain reason for usage archive?

Reply:
My initial thought was that if a user is archived, `attendables` should be deleted. So that when calling attendees, we can fetch users except for the deleted user. But I agree with you and I think we can call attendees without deleting it.
I will update it!